Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -239,7 +239,7 @@ class BitBuffer {
 
 ErrorOr<NonnullOwnPtr<BitBuffer>> BitBuffer::create(size_t width, size_t height)
 {
-    size_t pitch = ceil_div(width, 8ull);
```

Comment:
Commit message here leaves a bit to be desired.

"LibGfx: Explicitly cast literals to size_t in JBIG2Loader"
?

Reply:
Changed to that